project/ScalafixBuild.scala

```diff
@@ -229,7 +229,13 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
 
   override def buildSettings: Seq[Setting[_]] = List(
     // https://github.com/sbt/sbt/issues/5568#issuecomment-1094380636
-    versionPolicyIgnored += "com.lihaoyi" %% "pprint",
+    versionPolicyIgnored ++= Seq(
```

Comment:
This mimics https://github.com/scalacenter/scalafix/blob/16e9dce37c83c3b4796a26c4dc13567025a31730/build.sbt#L150-L157